spec/factories/topics.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :topic do
+    name { Faker::Name.name }
```

Comment:
Maybe you can change this for `Faker::Name.unique.name` to prevent possible accidental duplicates.